### PR TITLE
Add the shorthand for test all cases.

### PR DESCRIPTION
**Pull request by Shylock-Hg**

### CODE REVIEW

**Comment by jackwener** [comment]
LGTM

File: tests/Makefile [hunk 1]
```diff
@@ -94,6 +94,8 @@ slow-query: currdir
 tck: jobs slow-query
 	$(test_j) tck/steps/test_tck.py
 
+test-all: test tck
```
Comment (darionyaphet) [inline tests/Makefile:97]: Currently using `make tck` to start test cases. why support another command?

Comment (Shylock-Hg) [reply tests/Makefile:97]: Just for typing less.

Comment (darionyaphet) [reply tests/Makefile:97]: fine  it's an alias

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**